apps/src/applab/acemode/errorMapper.js

```diff
@@ -0,0 +1,35 @@
+var errorMap = [
+  {
+    original: /Assignment in conditional expression/,
+    replacement: "For conditionals, use the comparison operator (===) to check if two things are equal."
+  },
+  {
+    original: /(.*)\sis defined but never used./,
+    replacement: "$1 is defined, but it's not called in your program."
+  },
+  {
+    original: /(.*)\sis not defined./,
+    replacement: "$1 hasn't been declared yet."
+  }
+];
+
+/**
+ * Takes the results of a JSLint pass, and modifies the error text according to
+ * our mapping. Note this makes changes in place to the passed in results
+ * object.
+ */
+module.exports.processResults = function (results) {
+  results.data.forEach(function (item) {
+
+    errorMap.forEach(function (errorMapping) {
+      var old = errorMapping[0];
+      var replacement = errorMapping[1];
```

Comment:
Yep. From an earlier iteration. Ironic, considering the change I'm making. I bet jshint complains if I run it.